### PR TITLE
[WIP] Add Shape component

### PR DESCRIPTION
**Pull request by domcorvasce**
As requested in #19

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,6 +12,7 @@ test/coverage
 *.ntvs*
 *.njsproj
 *.sln
+*.swp
```
Comment (mariolamacchia) [inline .gitignore]: I'd suggest to add this to your global gitignore: `~/.gitignore`

File: docs/App.vue [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 <script>
 import 'semantic-ui-css/semantic.css';
+import 'semantic-ui-css/semantic.js';
```
Comment (mariolamacchia) [inline docs/App.vue]: We want to avoid adding other dependencies on top of this

File: .gitignore (hunk 1 above)
Comment (domcorvasce) [reply .gitignore]: Got it. I'll remove it.

File: docs/App.vue (hunk 2 above)
Comment (domcorvasce) [reply docs/App.vue]: What do you propose? As I pointed out to you in PM, shape component require jQuery and SUI.js to work.

Comment (mariolamacchia) [reply docs/App.vue]: Yes, we need to reimplement it. I'll try to help you

**Pull request closed without merging**